### PR TITLE
[SETUP] Add codepage-specific font substitutes

### PR DESCRIPTION
**Pull request by katahiromz**
## Purpose
Add codepage-specific font substitutes for i18n.
JIRA issue: [CORE-16266](https://jira.reactos.org/browse/CORE-16266)

### CODE REVIEW

**Comment by HBelusca** [comment]
Do we support such font substitutes in win32k?

**Comment by katahiromz** [comment]
@HBelusca We need some more fixes in win32k.

File: base/setup/lib/muifonts.h [hunk 1]
```diff
@@ -304,6 +439,11 @@ MUI_SUBFONT HindiFonts[] =
 {
     /*Font                       Substitute  */
     { L"Arial",                  L"Liberation Sans" },
+    { L"Arial Baltic,186",       L"Arial,186" },
+    { L"Arial CE,238",           L"Arial,238" },
+    { L"Arial CYR,204",          L"Arial,204" },
+    { L"Arial Greek,161",        L"Arial,161" },
+    { L"Arial TUR,162",          L"Arial,162" },
     { L"Courier",                L"FreeMono" },
     { L"Courier New",            L"FreeMono" },
```
Comment (SergeGautherie) [inline base/setup/lib/muifonts.h]: Is Hindi missing the 'Courier New' block intentionally?

**Comment by SergeGautherie** [comment]
Would it be possible/worth to support some kind of "common" list, to avoid such duplications?

**Comment by HBelusca** [comment]
@SergeGautherie : the definitive way would be to use a separate INI file instead of hardcoded lists within the installer.

**Comment by SergeGautherie** [comment]
> the definitive way

Ok. Is there a Jira issue about that improvement?

**Comment by HBelusca** [comment]
It is part of the "setup_improvements" plan, but there is no specific JIRA issue about that specific thing yet.

File: base/setup/lib/muifonts.h (hunk 1 above)
Comment (katahiromz) [reply base/setup/lib/muifonts.h]: Fixed.

**Pull request merged**